CMakeLists.txt

```diff
@@ -262,15 +263,32 @@ set_target_properties(${STLINK_LIB_SHARED} PROPERTIES
                         OUTPUT_NAME ${PROJECT_NAME}
                         )
 
+# Since we're not using __declspec(dllexport), WINDOWS_EXPORT_ALL_SYMBOLS is mandatory for MSVC.
+# This call will only affect MSVC, compilers/OSes ignore it.
+# TODO: Use dllexport on the desired functions
+set_target_properties(${STLINK_LIB_SHARED} PROPERTIES
+        WINDOWS_EXPORT_ALL_SYMBOLS ON
+)
+
 # Link shared library
 if (WIN32)
     target_link_libraries(${STLINK_LIB_SHARED} ${LIBUSB_LIBRARY} ${SSP_LIB} wsock32 ws2_32)
 else ()
     target_link_libraries(${STLINK_LIB_SHARED} ${LIBUSB_LIBRARY} ${SSP_LIB})
 endif()
 
-install(TARGETS ${STLINK_LIB_SHARED} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-
+# For windows, shared libraries go to `bin` and their implibs go to `lib`
```

Comment:
Documentaton: Replace "implibs" with "imported libraries (implibs)" in comment for clarification.

Reply:
On it! Will change it to "import Libraries (implibs)" since that's the full name for windows - these `.lib` files are not static libraries, they're _symbol containers_ and generally instruct MSVC linker how to link to the `.dll` .